docs/release/release_0_4_11.md

```diff
@@ -0,0 +1,219 @@
+- Fix windows 37 test (#2909)
+- Fix docstring, and type annotate. (#2912)
+- Don't raise exception when failing to save qt resources. (#2919)
+- Dix invalid yaml for docs workflow (#2920)
+- Fix use of `default_factory` in settings (#2930)
+- Close Qt progress bars when viewer is closed (#2931)
+- Degrade gracefully to default when colormap is not recognized (#2936)
+- Fix magicgui registration and circular imports (#2949)
+- Fix error in `Viewer.reset_view()` with vispy 0.7 (#2958)
+- Addressing case where argument to get_default_shape_type is empty list, addresses issue #2960 (#2961)
+- Fix nD anisotropic painting when scale is negative (#2966)
+- Ensure new height of expanded notification is larger than current height (#2981)
+- Gracefully handle properties with `object` dtype (#2986)
+- Fix scale decomp with composite (#2990)
+- Fix behavior of return/escape on preferences dialog to accept/cancel (#2998)
+- Fix EventedDict (#3011)
+- Use compression=('zlib', 1) for new tifffile (#3040)
+- Fix saving preferences (#3041)
+- Use non-deprecated colormap in viewer cmap test (#3043)
+- Fix Labels layer controls checkbox labels (#3046)
+- Fix Layer.affine assignment and broadcasting (#3056)
+- Fix problem with assigning affine with negative entries to  pyramids (#3088)
+- Fix stubgen and package stubs in wheel/sdists (#3105)
+- Fix opacity slider on shapes (#3109)
+- Fix empty points layer with color cycle (#3110)
+- Fix point deletion bug (#3119)
+- Fix for get_value() with mixed dims (#3121)
+- Fix settings reset breaking connections (creating a new instance of nested sub-models) (#3123)
+- Fix plane serialisation (#3143)
+- Bugfix in labels erasing (#3146)
+- Bug fix for undo history in 3D painting (#3154)
+- Revert `Points` `remove_selected` always overwriting `self._value` to `None` (#3165)
+- Fix window geometry loading bug, and make `ApplicationSettings` types more accurate (#3182)
+- Fix missing import in napari.__init__.pyi (#3183)
+- Fix incorrect window position storage (#3196)
+- Fix incorrect use of dims_order when 3D painting (#3202)
+
+
+## API Changes
+- Remove brush shape (#3047)
+- Enforce layer.metadata as dict (#3020)
+- Use enum objects in EventedModel (#3112)
+
+
+## UI Changes
+- Remove keybindings dialog from help menu (#3048)
+- Remove plugin sorter from plugin install dialog (#3069)
+- Update Labels layer keybindings to be more ergonomic (#3072)
+
+
+## Deprecations
+
+
+## Build Tools, Tests, Documentation, and other Tasks
+- Add imagecodecs to the bundle to open additional tiffs (#2895)
+- Make ordering of releases manual (#2921)
+- Add alister burt to team page (#2937)
+- Use briefcase 0.3.1 on all platforms (#2980)
+- Speedup one of the slowest test. (#2997)
+- Update plugin guide with references and instructions for napari-hub (#3055)
+- Add missing libraries in docker file and entrypoint (#3081)
+- Update documentation regarding the hub (#3091)
+- Fix stubgen and package stubs in wheel/sdists (#3105)
```

Comment:
this is actually a reasonably big quality of life improvement for those using napari in their IDE.  maybe feature?

Reply:
moved